Specs/createScene.js

```diff
@@ -16,9 +16,14 @@ define([
     "use strict";
 
     function createScene(options) {
+        options = defaultValue(options, {});
+
+        var canvas = defaultValue(options.canvas, createCanvas());
+        options.canvas = undefined;
```

Comment:
Can we add a comment about what is going on here, e.g., we don't want to clone the HTML canvas element.